### PR TITLE
feat: add `encoding` option (`options.encoding`)

### PR DESCRIPTION
**Pull request by michael-ciniawsky**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,13 +22,15 @@ module.exports = function(content) {
...
+  var encoding = options.encoding || 'base64';
+
   // No limits or limit more than content length
   if(!limit || content.length < limit) {
     if(typeof content === "string") {
       content = new Buffer(content);
     }
 
-    return "module.exports = " + JSON.stringify("data:" + (mimetype ? mimetype + ";" : "") + "base64," + content.toString("base64"));
+    return "module.exports = " + JSON.stringify("data:" + (mimetype ? mimetype + ";" : "") + options.encoding + "," + content.toString(options.encoding));
```
Comment (michael-ciniawsky) [inline index.js]: `"data:<mimetype>;<encoding>,<content...>"`

```
return "module.exports = " + JSON.stringify("data:" 
 + (mimetype ? mimetype + ";" : "") 
 + (options.encoding  ? options.encoding : (?*) )  + "," 
 + content.toString(options.encoding ? options.encoding : (?*) ));
```

`?*` => Discuss/Triage

**Comment by michael-ciniawsky** [comment]
Reference #28

**Comment by michael-ciniawsky** [comment]
Fixes #25 #28

File: index.js (hunk 1 above)
Comment (evilebottnawi) [reply index.js]: @michael-ciniawsky 
```js
return "module.exports = " + JSON.stringify("data:" + (mimetype ? mimetype + ";" : "") + encoding + "," + content.toString(encoding)
```

**Comment by michael-ciniawsky** [comment]
@evilebottnawi 👍  But I need to revisit this first and please take a look at #28 in conjunction aswell, when time :)

**Pull request closed without merging**